sdk/cosmos/azure-cosmos-spark_3-0_2-12/docs/configuration-reference.md

```diff
@@ -36,14 +36,16 @@ Configuration Reference:
 
 #### Schema Inference Config
 
-Note when running queries unless if are interested to get back the raw json payload 
-we recommend setting `spark.cosmos.read.inferSchemaEnabled` to be `true`. 
+When doing read operations, users can specify a custom schema or allow the connector to infer it. When no custom schema is provided and schema inference is not enabled, the connector will return raw json.
```

Comment:
please change, schema inference is enabled by default. no need to opt in by default.